ui/perfherder/compare/TableAverage.jsx

```diff
@@ -15,6 +15,15 @@ const TableAverage = ({ value, stddev, stddevpct, replicates, app }) => {
       .join(' ')} > ${formatNumber(displayNumber(stddev))} = ${formatNumber(
       displayNumber(stddevpct),
     )}% standard deviation)`;
+
+    const TOOLTIP_MAX_LENGTH = 250;
```

Comment:
can we move this to `ui/perfherder/perf-helpers/constants.js`, and camel-case it?